Fix SSBD mitigation for aarch64 (#2576)
firecracker-microvm/firecracker: docs/prod-host-setup.md

This will mitigate variants of Spectre side-channel issues such as
Speculative Store Bypass and SpectreNG.

On x86_64 systems, it can be enabled by adding the following Linux kernel boot
parameter:

```console
spec_store_bypass_disable=seccomp
```

which will apply SSB if seccomp is enabled by Firecracker.

> On aarch64 systems, it is done in the jailer [using the `prctl` interface][3].

Reviewer: ```suggestion
On aarch64 systems, it is enabled by Firecracker [using the `prctl` interface][3].
```

Reviewer: Please also update the commit message, which still mentions the jailer

Author: Done.
